rfi_file_monitor/utils.py

```diff
@@ -113,7 +113,7 @@ def update_from_dict(self, yaml_dict: dict):
                     else:
                         widget.set_text(value)
                 elif isinstance(widget, Gtk.ComboBoxText):
-                    widget.set_active("")
+                        widget.append(value)
```

Comment:
[`append()`](https://lazka.github.io/pgi-docs/index.html#Gtk-3.0/classes/ComboBoxText.html#Gtk.ComboBoxText.append) is used to extend the list of options to pick from in the combobox model: it doesn't set the currently selected value.

Reply:
There is no method to select the text on the ComboBox that I could see. I added the append method so that we could update whatever list we have with a value from a dictionary. It could be a catch all like “technique not listed” or “None”. Otherwise we can take the ComboBoxText out of the is instance list.

Reply:
There are actually three of them: [`set_active()`](https://lazka.github.io/pgi-docs/Gtk-3.0/classes/ComboBox.html#Gtk.ComboBox.set_active), [`set_active_id()`](https://lazka.github.io/pgi-docs/Gtk-3.0/classes/ComboBox.html#Gtk.ComboBox.set_active_id), [`set_active_iter()`](https://lazka.github.io/pgi-docs/Gtk-3.0/classes/ComboBox.html#Gtk.ComboBox.set_active_iter). 

The difficulty in using them is that you need to iterate over the model that backs the combobox to find the row whose contents match the value.

Reply:
in the last push I have an interation of this. It gets the model from the ComboBoxText, iterates through the rows to find which iteration matches the dictionary value, and then sets that as the active iter.

Reply:
This works when loading settings from a yaml file.